hermes/file_gen_test.go

```diff
@@ -0,0 +1,39 @@
+package hermes
+
+import(
+	//"fmt"
+	"testing"
+	"math/rand"
+	"strconv"
+	"time"
+	//"crypto/sha1"
+)
+
+
+func TestChecksum(t *testing.T){
+	var global_contents string = "jhfvjhdfjhfjjhjhdfvjvcvfjh"
+	var result_expected string = "_68f3caf439065824dcf75651c202e9f7c28ebf07"
+	file := HermesFile{}
+	file.contents = global_contents;
+	result := generate_file_checksum(file)
```

Comment:
Similar to the "want" from above, you can rename this one to "got".

Please do the same for the relevant variables in TestFileName.

```suggestion
	got := generate_file_checksum(file)
```